packages/api/src/tasks/checkAlerts.ts

```diff
@@ -52,7 +50,7 @@ const getAlerts = () =>
     team: ITeam;
     savedSearch?: EnhancedSavedSearch;
     dashboard?: IDashboard;
-  }>(['team', 'savedSearch', 'savedSearch.source', 'dashboard']);
+  }>(['team', 'savedSearch', 'dashboard']);
```

Comment:
`savedSearch.source` is still using in line 648: `const source = await Source.findById(alert.savedSearch.source);`

Just want double check if it is fine.

Reply:
his is fine. The `savedSearch` field in alert model is an id reference. I realized it couldn't be double-populated and that's why the source detail is fetched separately